catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/CatalogCacheSource.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.cache;
+
+import ddf.catalog.source.Source;
+
+/**
+ * Used by {@link ddf.catalog.plugin.PostFederatedQueryPlugin} and
+ * {@link ddf.catalog.plugin.PreFederatedQueryPlugin} to determine
+ * if a source is the cache.
+ */
+public interface CatalogCacheSource extends Source {
```

Comment:
I think it would be better to move this interface into `catalog-core-api`. It's probably fine where it is, but is a bit out of place. 

I'm also concerned that on some bundle refreshes you might end up with a class casting issue which would present itself by always returning false when you test `instanceof`. I'd want to test that with the interface in this bundle.